app/services/invitation_links_generator.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class InvitationLinksGenerator
```

Comment:
Why you created class, use module

Reply:
Why I need to use module ?

Reply:
@davinciRor added js validation for users sign up